From c36e43de9ec8b5a0b1ccd6dac6ea00082e2c8592 Mon Sep 17 00:00:00 2001
From: LuK1337 <priv.luk@gmail.com>
Date: Tue, 26 Jun 2018 15:50:31 +0200
Subject: [PATCH 1/2] Revert "Fix description path for libnqp61-jcop-kit and
 move to vendor"

* error: libnqnfc_nci_jni (native:platform) should not link to libnqp61-jcop-kit (native:vendor)

This reverts commit f48a62374e18c96ebe36c9373517ee2488e078db.

Change-Id: I247157670d2bff2a9e07df60ab0fae0e90006bf9
---
 p61-jcop-kit/Android.mk | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/p61-jcop-kit/Android.mk b/p61-jcop-kit/Android.mk
index 6e33068..a2a328d 100644
--- a/p61-jcop-kit/Android.mk
+++ b/p61-jcop-kit/Android.mk
@@ -10,7 +10,7 @@ endef
 LOCAL_PATH:= $(call my-dir)
 D_CFLAGS += -DNXP_LDR_SVC_VER_2=TRUE
 ######################################
-# Build shared vendor library libnqp61-jcop-kit.so for stack code.
+# Build shared library system/vendor/lib/libp61-jcop-kit.so for stack code.
 
 include $(CLEAR_VARS)
 LOCAL_PRELINK_MODULE := false
@@ -18,7 +18,6 @@ LOCAL_ARM_MODE := arm
 LOCAL_MODULE := libnqp61-jcop-kit
 LOCAL_MODULE_TAGS := optional
 LOCAL_MODULE_OWNER := nxp
-LOCAL_VENDOR_MODULE := true
 LOCAL_SHARED_LIBRARIES := libhardware_legacy libcutils liblog libdl libhardware
 LOCAL_CFLAGS := $(D_CFLAGS)
 LOCAL_C_INCLUDES := \
-- 
2.17.1

